iset.mm

```diff
@@ -27243,6 +27243,24 @@ practical reasons (to avoid having to prove sethood of ` A ` in every use
 -.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
 $)
 
+  ${
+    $d A x z $.  $d B x y z $.
+    $( Alternate definition of class difference.  Definition of relative set
+       complement in Section 2.3 of [Pierik], p. 10.  (Contributed by BJ and
+       Jim Kingdon, 16-Jun-2022.) $)
+    dfdif3 $p |- ( A \ B ) = { x e. A | A. y e. B x =/= y } $=
+      ( vz cdif cv wne wral crab wcel wn eldif simplbi eleq1a weq wi wex wa wal
+      necon3bd com12 ralrimiv simplbiim neeq1 ralbidv elrab sylanbrc a9e equcom
+      ssriv exbii mpbir df-ral con2b df-ne imbi2i notbid pm5.74i 3bitr4i exintr
+      eleq1 albii bitri sylbi mpi 19.41v sylib simprd a1i dfdif2 sseqtr4i eqssi
+      ss2rabi ) CDFZAGZBGZHZBDIZACJZEVOVTEGZVOKZWACKZWAVQHZBDIZWAVTKWBWCWADKZLZ
+      WACDMZNWBWCWGWEWHWGWDBDVQDKZWGWDWIWFWAVQVQDWAOUAUBUCUDVSWEAWACAEPVRWDBDVP
+      WAVQUEUFUGUHUKVTVPDKZLZACJVOVSWKACVSWKQVPCKVSABPZBRZWKVSWLWKSBRZWMWKSVSWM
+      WNWMBAPZBRBAUIWLWOBABUJULUMVSWLWKQZBTZWMWNQVSWIVRQZBTWQVRBDUNWRWPBWIWLLZQ
+      WLWILZQWRWPWIWLUOVRWSWIVPVQUPUQWLWKWTWLWJWIVPVQDVBURUSUTVCVDWLWKBVAVEVFWL
+      WKBVGVHVIVJVNACDVKVLVM $.
+  $}
+
```

Comment:
Shoud this theorem be added also to set.mm, shoudn't it?

Reply:
Since I don't really know the reason for writing this definition this way (like, is it a thing in multiple sources or just an oddity of this one paper?), I have trouble answering this question in terms of whether this is an important result and if so where. But if the thought process is "sure, why not?" then it becomes easier.

I've pushed up a change to add it to set.mm. The iset.mm proof works with only one change related to The Great Axiom Renumbering.

Reply:
And when I decided to minimize it that revealed a few more (unnecessary) differences between set.mm and iset.mm, so I fixed those up, which is the latest things I've pushed to this pull request.

Hopefully this isn't scope-creeping too badly.